frontend/client/views/settings/index.vue

```diff
@@ -260,6 +291,37 @@
       </div>
     </modal>
 
+    <!-- reset trigger token modal -->
+   <modal :visible="showResetPipelineTriggerTokenModal" class="modal-z-index" @close="close">
+      <div class="box pipeline-modal">
+        <div class="block pipeline-modal-content">
+          <collapse accordion is-fullwidth>
+            <collapse-item title="Reset Pipeline Trigger Token" selected>
+              <div class="pipeline-modal-content">
+                <p class="control has-icons-left" style="padding-bottom: 5px;">
+                  <!-- <input class="input is-medium input-bar" v-focus v-model="selectPipeline.name"
```

Comment:
If it's not needed we should remove it. Later nobody will remember why this was there. 😄

Reply:
Hah, thanks!